build/cli.js

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
```

Comment:
Found these explicit ones were required for Windows as well, otherwise it tried to use the JScript engine even with the explicit call to `node` from the commandline